### ISSUE TITLE
Project configuration not honored

### ISSUE THREAD

**Issue by jaraco** [thep0y/python-black#37]
Sometime recently, the package no longer seems to be honoring black config. Similar to #22, I have configuration in `pyproject.toml`:

```
[tool.black]
skip-string-normalization = true
```

Yet, when I save a file (with format on save set to smart), the file is reformatted and `'` is replaced by `"`.

Running a standalone black 23.1.0 against the same file results in no changes.

Invoking `python-black: Create Black Configuration File` opens the `pyproject.toml` with the above content.

I'm pretty sure this issue is new in the past 2-3 days as I frequently save Python files in projects with the reported configuration and I've only today noticed the issue.

**Comment by jaraco** [comment]
I uninstalled and reinstalled `python-black`, but behavior remains the same. Installed plugin version is 1.2.6.

**Comment by jaraco** [comment]
I can see that the setting is being read correctly:

```
 jaraco.collections main $ cat '~/Library/Application Support/Sublime Text 3/Log/python-black.log'
[INF] 14:30:40.455 python-black.commands:33 - versions: black=23.1.0, pathspec=0.11.0, platformdirs=3.0.0, tomli=2.0.1, appdirs=1.4.4
[INF] 14:30:41.367 python-black.commands:56 - use smart mode: True
[INF] 14:30:41.368 python-black.python_black.black:219 - configuration file used: /Users/jaraco/code/jaraco/jaraco.collections/pyproject.toml
[INF] 14:30:41.369 python-black.python_black.black:193 - configuration used: {'target_version': [], 'line_length': 88, 'string_normalization': True, 'is_pyi': False, 'skip_source_first_line': False, 'magic_trailing_comma': True, 'include': '(\\.pyi?|\\.ipynb)$', 'skip_string_normalization': True}
[INF] 14:31:36.700 python-black.commands:56 - use smart mode: True
[INF] 14:31:36.702 python-black.python_black.black:219 - configuration file used: /Users/jaraco/code/jaraco/jaraco.collections/pyproject.toml
[INF] 14:31:36.703 python-black.python_black.black:193 - configuration used: {'target_version': [], 'line_length': 88, 'string_normalization': True, 'is_pyi': False, 'skip_source_first_line': False, 'magic_trailing_comma': True, 'include': '(\\.pyi?|\\.ipynb)$', 'skip_string_normalization': True}
[INF] 14:41:36.913 python-black.commands:33 - versions: black=23.1.0, pathspec=0.11.0, platformdirs=3.0.0, tomli=2.0.1, appdirs=1.4.4
[INF] 14:41:43.462 python-black.commands:56 - use smart mode: True
[INF] 14:41:43.463 python-black.python_black.black:219 - configuration file used: /Users/jaraco/code/jaraco/jaraco.collections/pyproject.toml
[INF] 14:41:43.464 python-black.python_black.black:193 - configuration used: {'target_version': [], 'line_length': 88, 'string_normalization': True, 'is_pyi': False, 'skip_source_first_line': False, 'magic_trailing_comma': True, 'include': '(\\.pyi?|\\.ipynb)$', 'skip_string_normalization': True}
```

Oh, that's weird. There's both `skip_string_normalization: True` (expected) and `string_normalization: True` (not expected).

**Comment by thep0y** [comment]
Yes, the cause of this problem comes from https://github.com/thep0y/python-black/pull/31, I will fix it as soon as possible.

**Comment by returnchung** [comment]
Ah, sorry, it's my fault. I didn't notice that the options `string_normalization` and `magic_trailing_comma` is totally on the contrary with command lines and toml file. If anything needs me to do, please let me know.

**Comment by thep0y** [comment]
@returnchung Please fix this bug if you have time.

**Comment by returnchung** [comment]
Ok I can fix it quickly now.

**Comment by jaraco** [comment]
Thanks for the quick fix (released in 1.2.7). Since there was no comment here, I'd not realized it was fixed until I came back here to check. I've confirmed the fix is working.
